lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -79,15 +80,35 @@ class MockTimers {
     this.#timers.removeAt(position);
   }
 
-  async * #setIntervalPromisified(interval, startTime, ...args) {
+  async * #setIntervalPromisified(interval, startTime, options) {
     const context = this;
     const emitter = new EventEmitter();
+    if (options?.signal) {
+      try {
+        validateAbortSignal(options.signal, 'options.signal');
+      } catch (err) {
+        return PromiseReject(err);
+      }
```

Comment:
```suggestion
      validateAbortSignal(options.signal, 'options.signal');
```
this is already in an async function, the catch and rethrow is redundant

Reply:
Nice didn't know that! haha